### PR TITLE
hub -> gh

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Fall of from https://github.com/kubernetes/kubernetes/pull/97697

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
I don't know why `gh` is so greedy for permissions (see https://cli.github.com/manual/gh_auth_login). Updated PR

File: contributors/devel/sig-release/cherry-picks.md [hunk 1]
```diff
@@ -24,10 +24,11 @@ branches.
   kubernetes `origin` fork on GitHub and making a pull request against a
   configured remote `upstream` that tracks
   `https://github.com/kubernetes/kubernetes.git`, including `GITHUB_USER`.
-- Have `hub` installed, which is most easily installed via
-  `go get github.com/github/hub` assuming you have a standard golang
-  development environment.
-- A github personal access token which has permissions to access public repositories.
+- Have GitHub CLI (`gh`) installed following [installation instructions](https://github.com/cli/cli#installation).
+- A github personal access token which has permissions "repo" and "read:org".
+  Permissiosn are required for [gh auth login](https://cli.github.com/manual/gh_auth_login)
```
Comment (dims) [inline contributors/devel/sig-release/cherry-picks.md:29]: typo Sergey "Permissiosn"

Comment (SergeyKanzhelev) [reply contributors/devel/sig-release/cherry-picks.md]: fixed

**Review by saschagrunert — APPROVED** [review]

**Comment by saschagrunert** [comment]
@SergeyKanzhelev feel free to ping @kubernetes/sig-release-leads the next time. :)

**Pull request merged**